src/main/resources/templates/fragments/download-and-preview-single.html

```diff
@@ -2,17 +2,14 @@
 <html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
   <div th:fragment="download-and-preview-single(records_or_entries_or_items_string)">
     <p class="data-representations">
-      Download a copy as
+      Get this data as
+      <a th:href="${'/' + records_or_entries_or_items_string + '.json/' + (#strings.equals(records_or_entries_or_items_string, 'records') ? content.entryKey : (#strings.equals(records_or_entries_or_items_string, 'entries') ? content.entryNumber : content.itemHash))}">JSON</a>,
+      <a th:href="${'/' + records_or_entries_or_items_string + '.yaml/' + (#strings.equals(records_or_entries_or_items_string, 'records') ? content.entryKey : (#strings.equals(records_or_entries_or_items_string, 'entries') ? content.entryNumber : content.itemHash))}">YAML</a>,
```

Comment:
YAML no longer working (tested on the Item page).

Reply:
Looking